docs/recipes/Running-In-Production.md

```diff
@@ -111,7 +111,7 @@ Ultimately a Holocron Module Map is a JSON object of the following shape:
 }
 ```
 
-The `key` property is used to cache bust all module bundles. This works because its value is appended
+The `clientCacheRevision` property is optional and used to cache bust all module bundles. This works because its value is appended
```

Comment:
Highlight that it should only be used when client side cache busting is required